### DISCUSSION TITLE
How to expand a log placeholder, such as userId: [%X{user}]? Any ideas?

### DISCUSSION THREAD

**Discussion by hgaong** [apache/skywalking#11379] [Q&A] [answered]
<img width="1168" alt="image" src="https://github.com/apache/skywalking/assets/29444991/2092673a-0c9d-4b95-bf01-f1f3343ce2f4">

**Comment by wu-sheng** [comment]
Why does this become SkyWalking's concern to do this?

**Reply by wu-sheng** [reply]
This is a question for log framework. We don't have an interest in taking over this responsibility. And we should not.

**Reply by hgaong** [reply]
Our system aims to achieve fast problem localization, so it is convenient to directly display the userId in logs. Are you suggesting that we should consult the maintainers of apm-toolkit-logback regarding this issue?

**Reply by wu-sheng** [reply, marked as answer]
This is not our use case.
